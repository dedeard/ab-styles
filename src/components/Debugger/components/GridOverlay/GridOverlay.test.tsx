import React from 'react';
import { render } from '@testing-library/react';

import { GridOverlay } from '.';

describe('Grid Overlay', () => {
  it('should render with default styles', () => {
    const { container } = render(<GridOverlay />);

    expect(container.firstChild).toMatchSnapshot();
  });

  it('should render with visible styles when passed the visible prop', () => {
    const { container } = render(<GridOverlay visible />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
