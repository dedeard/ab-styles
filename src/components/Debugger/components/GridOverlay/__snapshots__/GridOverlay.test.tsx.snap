// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid Overlay should render with default styles 1`] = `
.grid-0 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10000;
  overflow: hidden;
  pointer-events: none;
  transform-origin: 50% 0%;
}

.grid-0 .e1g9m8r10 {
  height: 100%;
  opacity: 0;
  -webkit-transition: opacity 0.2s;
  transition: opacity 0.2s;
}

.grid-0 .elhoft0 {
  height: 100%;
  background: #0470F50D;
}

.grid-0 .e12n51c30 {
  height: 100%;
}

.grid-0 .e12n51c30>div {
  position: relative;
  width: 100%;
  height: 100%;
}

.grid-0 .e12n51c30>div::before,
.grid-0 .e12n51c30>div::after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #0470F50D;
}

.emotion-0 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10000;
  overflow: hidden;
  pointer-events: none;
  transform-origin: 50% 0%;
}

.emotion-0 .e1g9m8r10 {
  height: 100%;
  opacity: 0;
  -webkit-transition: opacity 0.2s;
  transition: opacity 0.2s;
}

.emotion-0 .elhoft0 {
  height: 100%;
  background: #0470F50D;
}

.emotion-0 .e12n51c30 {
  height: 100%;
}

.emotion-0 .e12n51c30>div {
  position: relative;
  width: 100%;
  height: 100%;
}

.emotion-0 .e12n51c30>div::before,
.emotion-0 .e12n51c30>div::after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #0470F50D;
}

<div
  class="emotion-0 emotion-1"
/>
`;

exports[`Grid Overlay should render with visible styles when passed the visible prop 1`] = `
.grid-0 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10000;
  overflow: hidden;
  pointer-events: none;
  transform-origin: 50% 0%;
}

.grid-0 .e1g9m8r10 {
  height: 100%;
  opacity: 1;
  -webkit-transition: opacity 0.2s;
  transition: opacity 0.2s;
}

.grid-0 .elhoft0 {
  height: 100%;
  background: #0470F50D;
}

.grid-0 .e12n51c30 {
  height: 100%;
}

.grid-0 .e12n51c30>div {
  position: relative;
  width: 100%;
  height: 100%;
}

.grid-0 .e12n51c30>div::before,
.grid-0 .e12n51c30>div::after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #0470F50D;
}

.emotion-0 {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  z-index: 10000;
  overflow: hidden;
  pointer-events: none;
  transform-origin: 50% 0%;
}

.emotion-0 .e1g9m8r10 {
  height: 100%;
  opacity: 1;
  -webkit-transition: opacity 0.2s;
  transition: opacity 0.2s;
}

.emotion-0 .elhoft0 {
  height: 100%;
  background: #0470F50D;
}

.emotion-0 .e12n51c30 {
  height: 100%;
}

.emotion-0 .e12n51c30>div {
  position: relative;
  width: 100%;
  height: 100%;
}

.emotion-0 .e12n51c30>div::before,
.emotion-0 .e12n51c30>div::after {
  content: '';
  position: absolute;
  top: 0;
  left: 0;
  width: 100%;
  height: 100%;
  background: #0470F50D;
}

<div
  class="emotion-0 emotion-1"
/>
`;
